aws/tailers/stack_event.go

```diff
@@ -0,0 +1,229 @@
+package awstailers
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"text/tabwriter"
+
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/fatih/color"
+	"github.com/wallix/awless/aws/services"
+)
+
+type stackEventTailer struct {
+	stackName        string
+	refresh          bool
+	watchDeployment  bool
+	refreshFrequency time.Duration
+	lastEventID      *string
+	nbEvents         int
+}
+
+type stackEvents []*cloudformation.StackEvent
+
+func coloredResourceStatus(str string) string {
+	switch {
+	case strings.HasSuffix(str, "_IN_PROGRESS"):
+		return color.New(color.FgYellow).SprintFunc()(str)
+	case strings.HasSuffix(str, "_COMPLETE"):
+		return color.New(color.FgGreen).SprintFunc()(str)
+	case strings.HasSuffix(str, "_FAILED"):
+		return color.New(color.FgRed).SprintFunc()(str)
+	default:
+		return str
+	}
+
+}
+
+func (e stackEvents) printReverse(w io.Writer) error {
```

Comment:
Same for this method (and again detail) that can go at the end of file to avoid cognitive effort on non interesting method each time we open this file.